### DISCUSSION TITLE
why can't I see my achievements in my github profile.

### DISCUSSION THREAD

**Discussion by Mrinalummadising** [community/community#75207] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

 here's my profile https://github.com/Mrinalummadising

**Comment by TankEngine-ish** [comment]
You can first check if they're not set to private by default - [this link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-achievements)

This should be ticked - 
![12313](https://github.com/community/community/assets/131184681/cb1ebc5f-ff91-4273-84ef-2faac86f0713)

**Comment by MohamedBoukthir** [comment]
just go to your profile setting and make it like this
![](https://user-images.githubusercontent.com/131184681/282244518-cb1ebc5f-ff91-4273-84ef-2faac86f0713.png)
 '
